webui/react/src/hooks/useModal/useModalCustomizeColumns.tsx

```diff
@@ -128,11 +136,11 @@ const useModalCustomizeColumns = ({
   const renderVisibleRow = useCallback(({ index, style }) => {
     const row = filteredVisibleColumns[index];
     return renderDraggableRow(
-      index,
       row,
+      index,
       style,
       () => makeHidden(row),
-      (itemIndex: number, newIndex:number) => switchRowOrder(itemIndex, newIndex),
+      (column: string, newColumnNeighbor: string) => switchRowOrder(column, newColumnNeighbor),
```

Comment:
`nit` i think this can just be `switchRowOrder`

Reply:
(also elsewhere `newColumnNeighbor` is referred to as `newNeighborColumn`, but this redundant if you make the first change)